README.md

```diff
@@ -78,8 +78,15 @@ function World() {
   };
 }
 
+// Use Before hook to perform async tasks
+function Before(callback) {
+  var server = require('http').createServer();
+  server.listen(8080, callback);
+}
+
 module.exports = function() {
   this.World = World;
+  this.Before = Before;
```

Comment:
This looks to me like you are replacing the `Before` hook, not adding your function to it? Shouldn't it be `this.Before(Before)` (or rename your `Before` function for clarity)? If I do this, my other `Before` listeners can't be added, because they call this function instead of the hook.